test/framework/integration_framework/fake_peer/network/ordering_gate_network_notifier.hpp

```diff
@@ -9,29 +9,21 @@
 #include <rxcpp/rx.hpp>
 
 #include "consensus/yac/transport/yac_network_interface.hpp"
+#include "framework/integration_framework/fake_peer/types.hpp"
 #include "network/ordering_gate_transport.hpp"
 
-namespace shared_model {
-  namespace interface {
-    class Proposal;
-  }
-}  // namespace shared_model
-
 namespace integration_framework {
   namespace fake_peer {
 
     class OgNetworkNotifier final
         : public iroha::network::OrderingGateNotification {
      public:
-      using Proposal = shared_model::interface::Proposal;
-      using ProposalPtr = std::shared_ptr<Proposal>;
-
-      void onProposal(ProposalPtr proposal) override;
+      void onProposal(OgProposalPtr proposal) override;
 
-      rxcpp::observable<ProposalPtr> get_observable();
+      rxcpp::observable<OgProposalPtr> get_observable();
```

Comment:
```suggestion
      rxcpp::observable<OgProposalPtr> getObservable();
```
Method name does not match the [style guide](https://google.github.io/styleguide/cppguide.html#Function_Names).